group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -420,6 +422,17 @@ public Group group(String groupId, long committedOffset) throws GroupIdNotFoundE
         return group;
     }
 
+    /**
+     * @return The GenericGroup List filtered by statesFilter and committedOffset.
+     */
```

Comment:
nit: Could we update the javadoc to full format? It would be great to document the arguments, etc. You have an example right below (`getOrMaybeCreateConsumerGroup`).